src/dots.ts

```diff
@@ -30,8 +30,8 @@ export class DotsNotation extends Notation {
     return `${this.dotify(exponent)}⣿${this.dotify(mantissa * 254)}`;
   }
 
-  private dotify(value: number, pad: boolean = false): string {
-    value = Math.round(value);
+  private dotify(rawValue: number, pad=false): string {
```

Comment:
Codacy found an issue: [expected parameter: 'pad' to have a typedef](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=5909622)

Reply:
Codacy finds an issue either way, and our general convention seems to be not to have a typedef.